src/main/java/plugily/projects/buildbattle/handlers/menu/registry/banner/Banner.java

```diff
@@ -64,14 +66,15 @@ public BannerPattern getLastPattern() {
   public ItemStack buildBanner() {
     ItemStack item = XMaterial.WHITE_BANNER.parseItem();
     BannerMeta meta = (BannerMeta) item.getItemMeta();
-    if(item instanceof org.bukkit.block.Banner) {
-      ((org.bukkit.block.Banner) item).setBaseColor(this.color);
-    } else {
-      meta.setBaseColor(this.color);
+
+    if(ServerVersion.Version.isCurrentEqualOrLower(ServerVersion.Version.v1_12_R1)) {
+      meta.setBaseColor(color);
```

Comment:
difference of item and meta location of setBaseColor now removed?

Reply:
No, the `ItemStack` can not be cast to `org.bukkit.block.Banner`, also `BannerMeta#addPattern` will handles the colours.